src/components/student/exam/questionsSet/QuestionsList.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="html">
+  <table class="table is-striped">
+    <tr
+      v-for="question in questions"
+      class="is-primary is-clickable">
+        <td
+          @click="selectQuestion"
+          :class="{'is-active': isActive(question.id)}"
+          :id="question.id"
+          :key="question.id">
+          {{question.question}}
```

Comment:
Just a reminder that it should eventually be changed to `question.content`.

Reply:
I will correct now all of those in the master, so I won't have to be remined of it again :)